tests/javaRosa.js

```diff
@@ -243,7 +247,7 @@ define([
 
             util.assertXmlEqual(
                 call('createXML'),
-                util.xmlines(TEST_XML_2),
```

Comment:
`xmlines` was a holdover from when these xml "files" were in strings at the bottom of this file. I moved those to files a while ago, but didn't realize they were using this helper